pkg/dashboard/server/gitContents.go

```diff
@@ -377,6 +377,13 @@ type envConfig struct {
 	DeployEvent     *dx.GitEvent
 }
 
+type DeploymentTemplate struct {
+	Name      string      `json:"name"`
+	Reference dx.Chart    `json:"reference"`
```

Comment:
Name and Reference carries the same information.

Try refactor it to use only Reference.